src/components/data/toolbar/TrashMenu.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @author sriram
+ *
+ * A button with menu items for DS trash operations.
+ *
+ */
+
+import React, { useState } from "react";
+import { useTranslation } from "i18n";
+
+import ids from "../ids";
+import styles from "../styles";
+
+import { build } from "@cyverse-de/ui-lib";
+import {
+    Button,
+    makeStyles,
+    Menu,
+    ListItemIcon,
+    ListItemText,
+    MenuItem,
+} from "@material-ui/core";
+
+import {
+    ArrowDropDown,
+    Delete,
+    DeleteForever,
+    RestoreFromTrash,
+    DeleteSweep,
+} from "@material-ui/icons";
```

Comment:
GitHub Actions complaining about a handful of different unused imports in this file, FYI